featuretools/tests/entityset_tests/test_timedelta.py

```diff
@@ -61,9 +62,9 @@ def test_delta_with_observations(es):
     large_delta = Timedelta(99999, 'observations', 'log')('customers',
                                                           instance_id=0,
                                                           entityset=es)
-    with pytest.raises(NotEnoughData):
+    with pytest.raises(NotEnoughData, match=''):
```

Comment:
no need to have match text here. In this case `NotEnoughData` is already specific enough